src/main/java/task/Event.java

```diff
@@ -0,0 +1,39 @@
+package task;
+
+import java.time.LocalDateTime;
+
+public class Event extends Task {
+    LocalDateTime startDate;
```

Comment:
Could this be protected or private?

Reply:
It was supposed to be private, seems like I did not write it in correctly.